### ISSUE TITLE
exit application in "Distraction Free Mode" - Menu bar is hidden and shortcuts are missing after restarting

### ISSUE THREAD

**Issue by Gregory-K** [pbek/QOwnNotes#1267]
#### Platform

qownnotes 19.8.5 on Windows 10 64bit

#### Expected behaviour

After restarting the application, all keyboard shortcuts and Menus should be working as before.

#### Actual behaviour

**In a rare occasion:**  
Having the Menu bar visible and entering "Distraction Free Mode",  
and then restarting the app,  
the Menu bar is hidden and some keyboard shortcuts are not working.

#### Steps to reproduce

- Open a note and use the "Distraction Free Mode" (further mentioned as DFM).
- Minimize to tray.
- Close/Exit application from tray icon (OR just shutdown the PC and boot again).
- Open/Start application again.
  - Application starts in DFM.
    - Shortcut to exit DFM is working only once, just to exit DFM.
    - Various other shortcuts are not working.  
      e.g. "`F4`, `F5`, `F6`, `Ctrl + ,`, and mostly all custom shortcuts"
    - Menu bar is missing.
      - `Ctrl + Shift + M` does not bring the Menu bar back and the shortcuts are still not working.
- Restart application after pressing `Ctrl + Shift + M`.
- Menu bar is now visible and all shortcuts are working.

#### Notes

*Always assuming running QOwnNotes running on Windows.*

Up until now, I am keeping the Menu bar visible. From previous experience, shortcuts wouldn't work if the Menu bar was hidden.

Well, that's not the case anymore (?) ... for the most of them. Some shortcuts will still be missing if the Menu bar is not visible from the start.

I don't know if the problem is only about custom shortcuts,  
e.g my custom shortcut to search notes in all folders is not working but the shortcut to search notes in current folder is working.  
Then again, `F4`, `F5`, `F6`, `Ctrl + ,` are predefined and some other custom shortcuts are still working (random? I don't know).

In any case, the problem with Menu bar visibility and keyboard shortcuts is known with QT (e.g. <https://github.com/pbek/QOwnNotes/issues/288>). I don't care so much, I just keep the Menu visible.

In this particular case the user keeps the Menu bar visible and suddenly it is lost. The shortcut to bring the Menu bar back is not working or there is no way for the user to know that it will work after an app restart.

On a general scope, it is good, for the new user, to be informed about hiding the Menu bar will cause some problems with his keyboard shortcuts (OR just disable the Menu hiding functionality until everything is working as expected?).

**Comment by pbek** [comment]
Thank you for reporting.
Can you please post the output from the debug settings (see issue template).
And is there any useful log output if you enable the log panel (see Window / Panels)? Please make sure you also enable debug logging.

**Comment by Gregory-K** [comment]
I tried debug logging and I saw nothing useful, but I can re-check and provide you the output.
As far, about the debug settings ... too much private info in there ... It will take a while to anonymize it.

**Comment by pbek** [comment]
Just delete everything that you don't want to post.

**Comment by pbek** [comment]
If you disable tray icon support and quit QON directly in DFM do you also get the same issue? I could reproduce it even w/o tray on Linux.

**Comment by Gregory-K** [comment]
Without "tray" it's even worse. It lost all "window panels" settings :)
*I don't know if having all the panel visibility settings configured while not having "tray" would do that.*

**Comment by Gregory-K** [comment]
QOwnNotes Debug Information
===========================

## General Info

**Current Date**: `Sat Aug 17 17:11:05 2019`
**Version**: `19.8.5`
**Build date**: `Aug 15 2019`
**Build number**: `4489`
**Platform**: `windows`
**Operating System**: `Windows 10 (10.0)`
**Build architecture**: `i386`
**Current architecture**: `x86_64`
**Release**: `AppVeyor`
**Qt Version (build)**: `5.13.0`
**Qt Version (runtime)**: `5.13.0`
**Portable mode**: `no`
**Settings path / key**: `\HKEY_CURRENT_USER\Software\PBE\QOwnNotes`
**Application database path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\QOwnNotes.sqlite`
**Application arguments**: `C:\Users\USER\AppData\Local\Notebook\QOwnNotes.exe`
**Qt Debug**: `0`
**Locale (system)**: `en_US`
**Locale (interface)**: *empty*
**Icon theme**: `breeze-qownnotes`
**Notes in current note folder**: `960`
**Calendar items**: `32`
**Enabled scripts**: `10`

## Server Info

**serverUrl**: `https://nextcloud.org`
**appIsValid**: `yes`
**notesPathExists**: `yes`
**serverVersion**: `17`
**appVersion**: *empty*

## Note folders

**currentNoteFolderId**: `2`

### Note folder `Notebook`

**id**: `2`
**isCurrent**: `yes`
**activeTagId**: `-1`
**localPath**: `F:\Notebook`
**remotePath**: `Notebook`
**isShowSubfolders**: `yes`
**isUseGit**: `no`
**activeNoteSubFolder name**: `SysAdmin`
**database file**: `F:\Notebook\notes.sqlite`

## Enabled scripts

### Script `Insert Table of Contents (TOC)`

**id**: `1`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\insert-toc\insert-toc.qml`
**variablesJson**: 
```
{
    "tocTitle": "## Table Of Contents"
}
```
**identifier**: `insert-toc`
**version**: `0.0.1`
**minAppVersion**: `17.06.2`

### Script `External note diff`

**id**: `2`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\external-note-diff\external-note-diff.qml`
**variablesJson**: 
```
{
    "executablePath": "C:\\Program Files\\Beyond Compare 4\\BCompare.exe"
}
```
**identifier**: `external-note-diff`
**version**: `0.0.1`
**minAppVersion**: `17.07.8`

### Script `Paste HTML as GitHub Markdown`

**id**: `5`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\paste-html-as-github-markdown\paste-html-as-github-markdown.qml`
**variablesJson**: 
```
{
    "pandocPath": "C:\\Users\\USER\\AppData\\Local\\Pandoc\\pandoc.exe"
}
```
**identifier**: `paste-html-as-github-markdown`
**version**: `0.0.1`
**minAppVersion**: `17.08.2`

### Script `Sort lines`

**id**: `6`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\sort-lines\sort-lines.qml`
**variablesJson**: *empty*
**identifier**: `sort-lines`
**version**: `0.0.1`
**minAppVersion**: `18.04.3`

### Script `Export notes as one HTML file`

**id**: `7`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\export-notes-as-one-html\export-notes-as-one-html.qml`
**variablesJson**: *empty*
**identifier**: `export-notes-as-one-html`
**version**: `1.0.0`
**minAppVersion**: `18.03.7`

### Script `Image preview`

**id**: `10`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\image-preview\image-preview.qml`
**variablesJson**: 
```
{
    "maxWidth": 640
}
```
**identifier**: `image-preview`
**version**: `1.0.0`
**minAppVersion**: `17.11.6`

### Script `Custom user interface styling`

**id**: `11`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\custom-stylesheet\custom-stylesheet.qml`
**variablesJson**: 
```
{
    "customStylesheet": "QTreeWidget {font-family: Segoe UI Semibold; font-size: 12px; background: #f7f7f7;}\nLineEdit {font-family: Segoe UI Semibold; font-size: 13px; background: #f7f7f7;}"
}
```
**identifier**: `custom-stylesheet`
**version**: `0.0.1`
**minAppVersion**: `17.05.7`

### Script `Journal entry`

**id**: `13`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\journal-entry\journal-entry.qml`
**variablesJson**: 
```
{
    "defaultFolder": "Journal",
    "defaultTags": "",
    "singleJournalPerDay": true
}
```
**identifier**: `journal-entry`
**version**: `1.2.0`
**minAppVersion**: `17.11.6`

### Script `Custom note preview styling`

**id**: `15`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\preview-styling\preview-styling.qml`
**variablesJson**: 
```
{
    "customStylesheet": "h2 {margin-top: 40px;border-bottom: 2px solid #eee;}\npre, code {white-space: pre-wrap;background-color: #f1f1f1;}"
}
```
**identifier**: `preview-styling`
**version**: `0.0.4`
**minAppVersion**: `17.05.7`

### Script `hash-headline`

**id**: `14`
**path**: `C:\Users\USER\AppData\Roaming\PBE\QOwnNotes\scripts\hash-headline\hash-headline.qml`
**variablesJson**: *empty*

## Settings

**AboutDialog/geometry** (QByteArray): `<binary data>`
**ActionDialog/geometry** (QByteArray): `<binary data>`
**ActiveNoteHistoryItem** (NoteHistoryItem): `<binary data>`
**Debug/fakeOldVersionNumber** (QString): `false`
**Debug/fileLogging** (QString): `false`
**DistractionFreeMode/editorWidthMode** (int): `3`
**DistractionFreeMode/isEnabled** (QString): `false`
**DistractionFreeMode/menuBarGeometry** (QByteArray): `<binary data>`
**DistractionFreeMode/menuBarHeight** (int): `21`
**DistractionFreeMode/menuBarVisible** (QString): `false`
**DistractionFreeMode/windowState** (QByteArray): `<binary data>`
**Editor/ColorSchemes** (QStringList): `EditorColorSchema-870fd4f5-c400-463f-b255-0761dcfe0af9, EditorColorSchema-c8eb716a-305f-4007-9643-fe26032375f3`
**Editor/CurrentSchemaKey** (QString): `EditorColorSchema-870fd4f5-c400-463f-b255-0761dcfe0af9`
**Editor/autoBracketClosing** (QString): `true`
**Editor/autoBracketRemoval** (QString): `true`
**Editor/disableCursorBlinking** (QString): `false`
**Editor/editorWidthInDFMOnly** (QString): `false`
**Editor/highlightCurrentLine** (QString): `true`
**Editor/vimMode** (QString): `false`

**EvernoteImportDialog/geometry** (QByteArray): `<binary data>`
**FileDialog/LastPath** (QString): `F:/Downloads`
**FileDialog/LastPath-ExternalEditor** (QString): `C:/Program Files/Sublime Text 3`
**FileDialog/LastPath-GitExecutable** (QString): `C:/Program Files/Git/bin`
**FileDialog/LastPath-InsertAttachment** (QString): `D:/`
**FileDialog/LastPath-NotePDFExport** (QString): `F:/Desktop`
**FileDialog/LastPath-SchemaExport** (QString): `F:/Desktop`
**FileDialog/LastPath-ScriptPath** (QString): `C:/Users/USER/AppData/Roaming/PBE/QOwnNotes/scripts/hash-headline`
**FileDialog/LastPath-ScriptSettingsFile-external-note-diff-executablePath** (QString): `C:/Program Files/Beyond Compare 4`
**FileDialog/LastPath-ScriptSettingsFile-paste-html-as-github-markdown-pandocPath** (QString): `C:/Users/USER/AppData/Local/Pandoc`
**FileDialog/LastPath-SettingsExport** (QString): `F:/Downloads`
**IssueAssistantDialog/geometry** (QByteArray): `<binary data>`
**LastUpdateCheck** (QDateTime): `2019-08-17T17:08:45.845`
**LinkDialog/geometry** (QByteArray): `<binary data>`
**LocalTrashDialog/geometry** (QByteArray): `<binary data>`
**MainWindow/geometry** (QByteArray): `<binary data>`
**MainWindow/mainToolBar.iconSize** (int): `16`
**MainWindow/menuBarGeometry** (QByteArray): `<binary data>`

**MainWindow/noteTextEdit.code.font** (QString): `Source Code Pro,11,-1,5,50,0,0,0,0,0,Regular`
**MainWindow/noteTextEdit.font** (QString): `Source Sans Pro,11,-1,5,50,0,0,0,0,0,Regular`
**MainWindow/noteTextView.code.font** (QString): `Source Code Pro,11,-1,5,50,0,0,0,0,0,Regular`
**MainWindow/noteTextView.font** (QString): `Source Sans Pro,11,-1,5,50,0,0,0,0,0,Regular`
**MainWindow/noteTextView.ignoreCodeFontSize** (QString): `true`
**MainWindow/noteTextView.rtl** (QString): `false`
**MainWindow/noteTextView.underline** (QString): `true`
**MainWindow/showNotesFromAllNoteSubFolders** (QString): `false`
**MessageBoxOverride/copy-notes** (int): `16384`
**MessageBoxOverride/local-trash-notes-removed** (int): `1024`
**MessageBoxOverride/local-trash-remove-notes** (int): `16384`
**MessageBoxOverride/move-notes** (int): `16384`
**MessageBoxOverride/note-replace-links** (int): `65536`
**MessageBoxOverride/readonly-mode** (int): `1024`
**MessageBoxOverride/remove-tags** (int): `16384`
**MessageBoxOverride/script-install-successful** (int): `1024`
**MessageBoxOverride/share-sync-information** (int): `1024`
**MessageBoxOverride/tag-notes** (int): `16384`
**NoteDialog/geometry** (QByteArray): `<binary data>`
**NoteHistory-2** (QVariantList): `<variant list with 200 item(s)>`
**NoteHistoryCurrentIndex-2** (int): `199`
**OrphanedImagesDialog/geometry** (QByteArray): `<binary data>`
**PasswordDialog/geometry** (QByteArray): `<binary data>`
**PiwikClientId** (QString): `<hidden>`
**Printer/NotePDFExportOrientation** (int): `0`
**Printer/NotePDFExportPageSize** (int): `4`
**ScriptRepositoryDialog/geometry** (QByteArray): `<binary data>`
**ScriptRepositoryDialog/mainSplitterState** (QByteArray): `<binary data>`
**SearchEngineId** (int): `2`
**SettingsDialog/geometry** (QByteArray): `<binary data>`
**SettingsDialog/mainSplitterState** (QByteArray): `<binary data>`
**ShareDialog/geometry** (QByteArray): `<binary data>`
**Shortcuts/MainWindow-actionAllow_note_editing** (QString): `F9`
**Shortcuts/MainWindow-actionAlphabetical** (QString): `Ctrl+Alt+.`
**Shortcuts/MainWindow-actionAutocomplete** (QString): `Shift+Space`
**Shortcuts/MainWindow-actionBy_date** (QString): `Ctrl+Alt+,`
**Shortcuts/MainWindow-actionDelete_line** (QString): `Ctrl+Shift+K`
**Shortcuts/MainWindow-actionEdit_encrypted_note** (QString): `Alt+Shift+E`
**Shortcuts/MainWindow-actionFind_action** (QString): `F1`
**Shortcuts/MainWindow-actionFind_notes_in_all_subfolders** (QString): `Ctrl+Shift+P`
**Shortcuts/MainWindow-actionInsert_attachment** (QString): `Alt+Shift+A`
**Shortcuts/MainWindow-actionOpen_List** (QString): `Ctrl+Alt+Shift+T`
**Shortcuts/MainWindow-actionReplace_in_current_note** (QString): `Ctrl+H`
**Shortcuts/MainWindow-actionSelect_note_folder** (QString): `F2`
**Shortcuts/MainWindow-actionShare_note** (QString): `Ctrl+Alt+Shift+S`
**Shortcuts/MainWindow-actionShow_local_trash** (QString): `Ctrl+Shift+Del`
**Shortcuts/MainWindow-actionToggle_distraction_free_mode** (QString): `F11`
**Shortcuts/MainWindow-actionUse_softwrap_in_note_editor** (QString): `Alt+W`
**Shortcuts/MainWindow-action_DuplicateText** (QString): `Ctrl+Shift+D`
**Shortcuts/MainWindow-action_Find_note** (QString): `Ctrl+P`
**Shortcuts/MainWindow-action_Find_text_in_note** (QString): `Ctrl+F`
**Shortcuts/MainWindow-action_Increase_note_text_size** (QString): `Ctrl+=`
**Shortcuts/MainWindow-action_Open_note_in_external_editor** (QString): `Ctrl+O`
**Shortcuts/MainWindow-action_Print_note_markdown** (QString): `Ctrl+Alt+F8`
**Shortcuts/MainWindow-action_Print_note_text** (QString): `Ctrl+F8`
**Shortcuts/MainWindow-action_Reload_note_folder** (QString): `Ctrl+R`
**Shortcuts/MainWindow-action_Reset_note_text_size** (QString): `Ctrl+0`
**Shortcuts/MainWindow-action_new_tag** (QString): `Ctrl+Shift+T`
**Shortcuts/MainWindow-customAction_addMultipleTags** (QString): `Ctrl+Shift+Y`
**Shortcuts/MainWindow-customAction_externalNoteDiff** (QString): `Ctrl+Alt+D`
**Shortcuts/MainWindow-customAction_html2Markdown** (QString): `Ctrl+Shift+G`
**Shortcuts/MainWindow-customAction_journalEntry** (QString): `Ctrl+D`
**Shortcuts/MainWindow-noteFolder-2** (QString): `Ctrl+2`
**Shortcuts/MainWindow-restoreWorkspace-57bffb7a-77f8-469d-8e98-2bfe64ec5f3b** (QString): `F7`
**Shortcuts/MainWindow-togglePanel-noteNavigationDockWidget** (QString): `Ctrl+G`
**Shortcuts/MainWindow-togglePanel-notePreviewDockWidget** (QString): `F12`
**ShowSystemTray** (QString): `true`
**StartHidden** (QString): `false`
**TableDialog/geometry** (QByteArray): `<binary data>`
**TodoDialog/geometry** (QByteArray): `<binary data>`
**TodoDialog/mainSplitterState** (QByteArray): `<binary data>`
**TodoDialog/showCompletedItems** (int): `2`
**TodoDialog/todoListSelectorSelectedItem** (QString): `@WORK`
**TrashDialog/geometry** (QByteArray): `<binary data>`
**UpdateDialog/geometry** (QByteArray): `<binary data>`
**WelcomeDialog/geometry** (QByteArray): `<binary data>`
**acceptAllExternalModifications** (QString): `true`
**allowDifferentNoteFileName** (QString): `false`
**allowNoteEditing** (QString): `true`
**allowOnlyOneAppInstance** (QString): `true`
**appMetrics/disableAppHeartbeat** (QString): `true`
**appMetrics/disableTracking** (QString): `true`
**appMetrics/notificationShown** (QString): `true`
**automaticNoteFolderDatabaseClosing** (QString): `true`
**closeTodoListAfterSave** (QString): `false`
**cryptoKey** (qlonglong): `<hidden>`
**currentNoteFolderId** (int): `2`
**currentWorkspace** (QString): `7199cad9-e44b-43a7-b24c-7f547ac3bfbf`
**cursorWidth** (int): `2`
**customNoteFileExtensionList** (QStringList): `tasks`
**darkMode** (QString): `false`
**darkModeColors** (QString): `false`
**darkModeIconTheme** (QString): `false`
**darkModeTrayIcon** (QString): `true`
**defaultNoteFileExtension** (QString): `md`
**demoNotesCreated** (QString): `true`
**disableAutomaticUpdateDialog** (QString): `true`
**disableSavedSearchesAutoCompletion** (QString): `true`
**dockWasInitializedOnce** (QString): `true`
**enableSocketServer** (QString): `false`
**externalEditorPath** (QString): `C:/Program Files/Sublime Text 3/sublime_text.exe`
**fullyHighlightedBlockquotes** (QString): `true`
**gitCommitInterval** (int): `28800`
**gitExecutablePath** (QString): *empty*
**gitLogCommand** (QString): *empty*
**guiFirstRunInit** (QString): `true`
**ignoreAllExternalModifications** (QString): `false`
**ignoreAllExternalNoteFolderChanges** (QString): `false`
**ignoreNoteSubFolders** (QString): `^\.`
**imageScaleDown** (QString): `false`
**imageScaleDownMaximumHeight** (int): `1024`
**imageScaleDownMaximumWidth** (int): `1024`
**insertTimeFormat** (QString): `yyyy.MM.dd hh:mm`
**interfaceFontSize** (int): `11`
**interfaceLanguage** (QString): *empty*
**internalIconTheme** (QString): `true`
**itemHeight** (int): `18`
**localTrash/autoCleanupDays** (int): `15`
**localTrash/autoCleanupEnabled** (QString): `true`
**localTrash/supportEnabled** (QString): `true`
**localTrashSplitterSizes** (QByteArray): `<binary data>`
**markdownHighlightingEnabled** (QString): `true`
**navigationPanelHideSearch** (QString): `false`
**networking/ignoreSSLErrors** (QString): `false`
**networking/proxyType** (int): `2`
**newNoteAskHeadline** (QString): `false`
**noteEditIsCentralWidget** (QString): `true`
**noteListPreview** (QString): `false`
**noteSaveIntervalTime** (int): `10`
**noteSubfoldersPanelDisplayAsFullTree** (QString): `true`
**noteSubfoldersPanelHideSearch** (QString): `true`
**noteSubfoldersPanelOrder** (int): `0`
**noteSubfoldersPanelShowFullPath** (QString): `false`
**noteSubfoldersPanelShowNotesRecursively** (QString): `false`
**noteSubfoldersPanelShowRootFolderName** (QString): `true`
**noteSubfoldersPanelSort** (int): `0`
**notesPanelOrder** (int): `0`
**notesPanelSort** (int): `1`
**notesPath** (QString): `D:/Documents/Notebook`
**notifyAllExternalModifications** (QString): `false`
**overrideInterfaceFontSize** (QString): `false`
**ownCloud/password** (QString): `<hidden>`
**ownCloud/serverUrl** (QString): `https://nextcloud.org`
**ownCloud/supportEnabled** (QString): `true`
**ownCloud/todoCalendarBackend** (int): `3`

**showMenuBar** (QString): `true`
**showStatusBar** (QString): `true`
**skipVersion** (QString): `19.8.0`
**systemIconTheme** (QString): `false`
**tagWasAddedToNote** (QString): `true`
**taggingShowNotesRecursively** (QString): `false`
**tagsPanelHideSearch** (QString): `true`
**tagsPanelOrder** (int): `0`
**tagsPanelSort** (int): `1`
**toolbar/1/items** (QStringList): `actionAlphabetical, actionBy_date, , action_New_note, action_Find_note, action_Remove_note, action_Open_note_in_external_editor, actionShow_local_trash, actionAllow_note_editing, , action_Back_in_note_history, action_Forward_in_note_history, , action_Find_text_in_note, actionReplace_in_current_note`
**toolbar/1/name** (QString): `mainToolBar`
**toolbar/1/title** (QString): `main toolbar`
**toolbar/2/items** (QStringList): `actionFormat_text_bold, actionFormat_text_italic, actionStrike_out_text, actionInsert_code_block, actionInsert_block_quote`
**toolbar/2/name** (QString): `formattingToolbar`
**toolbar/2/title** (QString): `formatting toolbar`
**toolbar/3/items** (QStringList): `actionInsert_Link_to_note, actionInsert_image, actionInsert_current_time`
**toolbar/3/name** (QString): `insertingToolbar`
**toolbar/3/title** (QString): `inserting toolbar`
**toolbar/4/items** (QStringList): `action_Encrypt_note, actionEdit_encrypted_note, actionDecrypt_note`
**toolbar/4/name** (QString): `encryptionToolbar`
**toolbar/4/title** (QString): `encryption toolbar`
**toolbar/5/items** (QStringList): `actionWorkspaceComboBox, actionStore_as_new_workspace, actionRemove_current_workspace, actionRename_current_workspace, actionSwitch_to_previous_workspace, actionUnlock_panels, , actionToggle_distraction_free_mode, action_Increase_note_text_size, action_Decrease_note_text_size, action_Reset_note_text_size`
**toolbar/5/name** (QString): `windowToolbar`
**toolbar/5/title** (QString): `window toolbar`
**toolbar/6/items** (QStringList): `action_Quit`
**toolbar/6/name** (QString): `quitToolbar`
**toolbar/6/title** (QString): `quit toolbar`
**toolbar/size** (int): `6`
**trashSplitterSizes** (QByteArray): `<binary data>`
**useSoftWrapInNoteEditor** (QString): `true`
**useUNIXNewline** (QString): `true`
**webSocketServerService/bookmarksNoteName** (QString): `Bookmarks`
**webSocketServerService/bookmarksTag** (QString): `bookmarks`
**webSocketServerService/port** (int): `22222`
**workspace-7199cad9-e44b-43a7-b24c-7f547ac3bfbf/name** (QString): `minimal`
**workspace-7199cad9-e44b-43a7-b24c-7f547ac3bfbf/noteSubFolderDockWidgetVisible** (QString): `true`
**workspace-7199cad9-e44b-43a7-b24c-7f547ac3bfbf/windowState** (QByteArray): `<binary data>`
**workspaces** (QStringList): `7199cad9-e44b-43a7-b24c-7f547ac3bfbf`

## System environment

**ALLUSERSPROFILE**: `C:\ProgramData`
**APPDATA**: `C:\Users\USER\AppData\Roaming`
**ChocolateyInstall**: `C:\ProgramData\chocolatey`

**ChocolateyToolsLocation**: `C:\tools`
**CLASSPATH**: `.;C:\Program Files (x86)\QuickTime\QTSystem\QTJava.zip`
**CommonProgramFiles**: `C:\Program Files (x86)\Common Files`
**CommonProgramFiles(x86)**: `C:\Program Files (x86)\Common Files`
**CommonProgramW6432**: `C:\Program Files\Common Files`
**COMPUTERNAME**: `WORKSTATION`
**ComSpec**: `C:\Windows\system32\cmd.exe`
**C_EM64T_REDIST11**: `C:\Program Files (x86)\Common Files\Intel\Shared Files\cpp\`
**DriverData**: `C:\Windows\System32\Drivers\DriverData`
**FPS_BROWSER_APP_PROFILE_STRING**: `Internet Explorer`
**FPS_BROWSER_USER_PROFILE_STRING**: `Default`
**GOPATH**: `D:\GO`
**GYP_MSVS_VERSION**: `2015`
**HOMEDRIVE**: `C:`
**HOMEPATH**: `\Users\USER`
**LOCALAPPDATA**: `C:\Users\USER\AppData\Local`
**LOGONSERVER**: `\\WORKSTATION`
**NUMBER_OF_PROCESSORS**: `8`

**OPENSSL_CONF**: `C:\Users\USER\scoop\apps\openssl\current\bin\cnf\openssl.cnf`
**OS**: `Windows_NT`
**Path**: `C:\tools\python2\;C:\tools\python2\Scripts;C:\tools\python3\Scripts\;C:\tools\python3\;C:\tools\php7;C:\ProgramData\Boxstarter;C:\Program Files (x86)\Common Files\Intel\Shared Files\cpp\bin\Intel64;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\ProgramData\Oracle\Java\javapath;C:\Program Files\ImageMagick;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\QuickTime\QTSystem\;C:\ProgramData\chocolatey\bin;C:\Program Files\7-Zip;C:\Program Files (x86)\UniConvertor\;C:\Program Files (x86)\UniConvertor\DLLs;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git\cmd;C:\Program Files\Microsoft VS Code\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\tools\lxrunoffline;C:\Go\bin;C:\Program Files\nodejs\;C:\Users\USER\scoop\apps\openssl\current\bin;C:\Users\USER\scoop\shims;C:\Users\USER\AppData\Local\Microsoft\WindowsApps;C:\Users\USER\AppData\Local\Pandoc\;F:\xtraPrograms\ADB-Fastboot;C:\Program Files\Microsoft VS Code\bin;C:\Program Files\Oracle\VirtualBox;C:\Users\USER\AppData\Roaming\Composer\vendor\bin;C:\Users\USER\go\bin;C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin;C:\Users\USER\AppData\Local\Microsoft\WindowsApps;C:\Users\USER\go\bin;C:\Users\USER\AppData\Roaming\npm`
**PATHEXT**: `.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW`
**PROCESSOR_ARCHITECTURE**: `x86`
**PROCESSOR_ARCHITEW6432**: `AMD64`
**PROCESSOR_IDENTIFIER**: `Intel64 Family 6 Model 60 Stepping 3, GenuineIntel`
**PROCESSOR_LEVEL**: `6`
**PROCESSOR_REVISION**: `3c03`
**ProgramData**: `C:\ProgramData`
**ProgramFiles**: `C:\Program Files (x86)`
**ProgramFiles(x86)**: `C:\Program Files (x86)`
**ProgramW6432**: `C:\Program Files`
**PSModulePath**: `C:\Users\USER\Documents\WindowsPowerShell\Modules`
**PUBLIC**: `C:\Users\Public`
**QTJAVA**: `C:\Program Files (x86)\QuickTime\QTSystem\QTJava.zip`
**QtMsBuild**: `C:\Users\USER\AppData\Local\QtMsBuild`
**SESSIONNAME**: `Console`
**SystemDrive**: `C:`
**SystemRoot**: `C:\Windows`
**TEMP**: `C:\Users\USER\AppData\Local\Temp`
**TMP**: `C:\Users\USER\AppData\Local\Temp`
**USERDOMAIN**: `WORKSTATION`
**USERDOMAIN_ROAMINGPROFILE**: `WORKSTATION`
**USERNAME**: `USER`
**USERPROFILE**: `C:\Users\USER`
**VBOX_MSI_INSTALL_PATH**: `C:\Program Files\Oracle\VirtualBox\`
**windir**: `C:\Windows`

**Comment by Gregory-K** [comment]
The `QOwnNotes.log` 

Start and Entering DFM
```
[Aug 17 18:08:01] [debug]: toggleDistractionFreeMode  - 'isInDistractionFreeMode':  false

**Comment by pbek** [comment]
It really is strange. This all worked with previous (Qt versions?) versions. I can't get a hold on it, seems like the menu bar settings are leaking somewhere. They should not get stored in DFM.

**Comment by Gregory-K** [comment]
... some random app crashes also

just mentioning it, I don't if it's relevant or a total different situation

```
<ProblemSignatures>
<EventType>APPCRASH</EventType>
<Parameter0>QOwnNotes.exe</Parameter0>
<Parameter1>0.0.0.0</Parameter1>
<Parameter2>5d5532c6</Parameter2>
<Parameter3>Qt5Core.dll</Parameter3>
<Parameter4>5.13.0.0</Parameter4>
<Parameter5>5d0274f3</Parameter5>
<Parameter6>c0000005</Parameter6>
<Parameter7>001eb0e3</Parameter7>
</ProblemSignatures>

<ProblemSignatures>
<EventType>APPCRASH</EventType>
<Parameter0>QOwnNotes.exe</Parameter0>
<Parameter1>0.0.0.0</Parameter1>
<Parameter2>5d5532c6</Parameter2>
<Parameter3>StackHash_2beb</Parameter3>
<Parameter4>0.0.0.0</Parameter4>
<Parameter5>00000000</Parameter5>
<Parameter6>c0000005</Parameter6>
<Parameter7>PCH_C2_FROM_ntdll+0x0007224C</Parameter7>
</ProblemSignatures>
```

**Comment by pbek** [comment]
Are you on 19.8.5 still or on 19.8.6?

**Comment by Gregory-K** [comment]
OK, probably the change to disable and re-enable the tray icon (some weird things happened with the workspace settings) OR just the 19.8.5 version was the problem.
After updating to 19.8.6 and rebooting, the random crashes are gone.

The issue's described bugs are still present.

**Comment by pbek** [comment]
I think I got it...

## 19.8.7
- fixed a menu bar problem when exiting the application in distraction free mode

**Comment by pbek** [comment]
There now is a new release, could you please test it and report if it works for you?

**Comment by Gregory-K** [comment]
OK, as soon as it becomes available (update check is still returning 19.8.6 as latest) I will test it.

**Comment by Gregory-K** [comment]
OK, FIXED with 19.8.7 release

**Comment by pbek** [comment]
Awesome, thank you for reporting and testing!
